src/server/main.py

```diff
@@ -13,6 +15,18 @@
 from .endpoints.admin import bp as admin_bp, enable_admin
 from ._limiter import limiter, apply_limit
 
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
+if SENTRY_DSN:
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        traces_sample_rate=float(os.environ.get('SENTRY_TRACES_SAMPLE_RATE', 1.0)),
+        profiles_sample_rate=float(os.environ.get('SENTRY_PROFILES_SAMPLE_RATE', 1.0)),
+        environment=str(os.environ.get('SENTRY_ENVIRONMENT', 'development')),
+        debug=str(os.environ.get('SENTRY_DEBUG', 'True')),
+        attach_stacktrace=str(os.environ.get('SENTRY_ATTACH_STACKTRACE', 'True'))
```

Comment:
do these options expressly accept strings? if they expect bools, any string except the empty string may evaluate to `True`...

also, is there a performance penalty with the `debug` option?  if so, maybe make it default to False and set the environment variable to True in our Makefile?  otherwise prod needs the option, and it feels better to me to make the development setup responsible for that.  ¯\\\_(ツ)_/¯  your call